packages/ckeditor5-dev-docs/lib/index.js

```diff
@@ -20,36 +18,32 @@ module.exports = {
  * @param {Object} config
  * @param {Array.<String>} config.sourceFiles Glob pattern with source files.
  * @param {String} config.readmePath Path to `README.md`.
- * @param {String} config.destinationPath Path under which documentation should be generated.
  * @returns {Promise}
  */
 function build( config ) {
-	return tools.clean( config.destinationPath, '.' )
-		.then( () => {
-			const sourceFiles = [
-				config.readmePath,
-				...config.sourceFiles
-			];
+	const sourceFiles = [
+		config.readmePath,
+		...config.sourceFiles
+	];
 
-			const jsDocConfig = {
-				opts: {
-					encoding: 'utf8',
-					destination: path.join( config.destinationPath, 'api' ),
-					recurse: true,
-					access: 'all'
-				},
-				plugins: [
-					'node_modules/jsdoc/plugins/markdown',
-					require.resolve( '@ckeditor/jsdoc-plugins/lib/export-fixer/export-fixer' ),
-					require.resolve( '@ckeditor/jsdoc-plugins/lib/longname-fixer/longname-fixer' ),
-					require.resolve( '@ckeditor/jsdoc-plugins/lib/validator/validator' ),
-					require.resolve( '@ckeditor/jsdoc-plugins/lib/utils/doclet-logger' )
-				]
-			};
+	const jsDocConfig = {
+		opts: {
+			encoding: 'utf8',
+			recurse: true,
+			access: 'all',
+			template: 'templates/silent'
```

Comment:
- `encoding`is not needed as `utf8` is default value

- `recurse` is not needed because `docs` task doesn't traverse directories (https://github.com/ckeditor/ckeditor5/blob/master/gulpfile.js#L21-L31)

- `access` should stay as `all` because by default `private` symbols are skipped by jsdoc

- `template` should stay cause `silent` isn't default